lib/va1010_forms/utils.rb

```diff
@@ -51,7 +51,19 @@ def submission_body(formatted_form)
 
     def log_payload_info(formatted_form, submission_body)
       form_name = formatted_form.dig('va:form', 'va:formIdentifier', 'va:value')
-      attachment_count = formatted_form.dig('va:form', 'va:attachments')&.length || 0
+      attachments = formatted_form.dig('va:form', 'va:attachments')
+      attachment_count = attachments&.length || 0
+      # Log the attachment sizes in descending order
+      if attachment_count.positive?
```

Comment:
Would this be a little cleaner if we pulled it out to another method for logging the attachment sizes? Not a huge deal, this method is just getting kind of chonky for what it is.

Reply:
I think it's okay because the attachment sizes still fall under `payload_info`.